django_extensions/management/modelviz.py

```diff
@@ -210,7 +211,7 @@ def add_relation(field, extras=""):
                     related_query_name = field.related_query_name()
                     if verbose_names and related_query_name.islower():
                         related_query_name = related_query_name.replace('_', ' ').capitalize()
-                    label = '{} ({})'.format(label, force_bytes(related_query_name))
+                    label = '{} ({})'.format(label, force_text(related_query_name))
```

Comment:
probably this needs to become:
```
label = u'{} ({})'.format(label, force_text(related_query_name))
```
or
```
label = force_text('{} ({})').format(label, force_text(related_query_name))
```

or probably in the beginning of the file to use `from __future__ import unicode_literals`